js_modules/dagster-ui/packages/ui-core/src/instance/InstanceBackfills.tsx

```diff
@@ -103,6 +106,41 @@ export const InstanceBackfills = () => {
   const isDaemonHealthy = useIsBackfillDaemonHealthy();
   const refreshState = useQueryRefreshAtInterval(queryResult, FIFTEEN_SECONDS);
   const {loading, data} = queryResult;
+  const [didDismissBackfillPageAlert, setDidDismissBackfillPageAlert] =
+    useStateWithStorage<boolean>('new_backfill_location_alert', (json) => !!json);
+
+  interface AlertProps {
+    setDidDismissBackfillPageAlert: (didDismissBackfillPageAlert: boolean) => void;
+  }
+
+  const BackfillPageDeprecationAlert = (props: AlertProps) => {
+    const {setDidDismissBackfillPageAlert} = props;
+
+    return (
+      <Box padding={8} border="top-and-bottom">
+        <Alert
+          title={
+            <>
+              <span>Backfills are moving:</span>
+              <span style={{fontWeight: 'normal'}}>
+                {' '}
+                We&apos;re incorporating backfills into the <Link to="/runs/">Runs</Link> page to
```

Comment:
could use some wordsmithing - working on that now

Reply:
We're incorporating backfills into the Runs page to simplify the UI and provide one page to see what is running. 

We're incorporating backfills into the Runs page to simplify the UI. The Backfills page will be removed in a future release.

Reply:
Maybe "to see **everything that** is running. "

Reply:
>simplify the UI
unify the UI?

Reply:
a slight edit option: "We're incorporating backfills into the Runs page to provide a single page to see all of your executions. The Backfills page will be removed in a future release."

reason being that no all things we show on these pages are in progress (i.e. running)